cli/objects/configuration.py

```diff
@@ -88,3 +76,42 @@ def _get_rules(self, rules_json: list[dict[Any, Any]]) -> Optional[list[Rule]]:
                 "Firewatch config is empty, please populate the configuration and try again.",
             )
             exit(1)
+
+    def _get_default_jira_project(self) -> str:
+        """
+        Used to get the default jira project from the FIREWATCH_DEFAULT_JIRA_PROJECT environment variable
+        :return: The string of the default environment variable.
+        """
+
+        default_project = os.getenv("FIREWATCH_DEFAULT_JIRA_PROJECT")
+
+        if default_project:
+            return default_project
+        else:
+            self.logger.error(
+                "Environment variable $FIREWATCH_DEFAULT_JIRA_PROJECT is not set, please set the variable and try again.",
+            )
+            exit(1)
+
+    def _get_config_data(self, config_file_path: Optional[str]) -> str:
+        """"""
```

Comment:
Just an FYI, missing docstring here.

Reply:
Fixed in [dc163bd](https://github.com/CSPI-QE/firewatch/pull/23/commits/dc163bdcb6400f16ab207376c34d4f04f83193e4). Thanks!